examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.zap

```diff
@@ -1136,6 +1136,22 @@
               "maxInterval": 65534,
               "reportableChange": 0
             }
+          ],
+          "events": [
+            {
+              "name": "AccessControlEntryChanged",
+              "code": 0,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1
+            },
+            {
+              "name": "AccessControlExtensionChanged",
```

Comment:
Per discussion at https://csamembers.slack.com/archives/G014G30SVV0/p1675695871232519?thread_ts=1675477777.040329&cid=G014G30SVV0 this should not be here: EventList should only include events the implementation might generate, and since this app does not implement the `Extension` attribute it can never generate this event.

Again, we should probably be carefully auditing every example app to make sure the events we enable match the spec...

Reply:
Will update. 

Should the spec specify that this Event is Mandatory only is `Extension` attribute is enabled?

Reply:
#6258 to follow up